### DISCUSSION TITLE
Not able to get features (layer, source, properties) onClick in v7

### DISCUSSION THREAD

**Discussion by Shashank1003** [uber/react-map-gl#1830] [Q&A] [answered]
I want to get the layer information and feature properties on click of any layer. Earlier I was doing it with event.features but after the upgrade, I am not getting features in the onClick event.
current onClick event object contains lngLat, originalEvent, point, and target. Please help me if I am missing something.

**Comment by Pessimistress** [comment, marked as answer]
https://visgl.github.io/react-map-gl/docs/api-reference/map#interactivelayerids

**Reply by Shashank1003** [reply]
It worked. Thank you for your help.

**Reply by bernarden** [reply]
@Pessimistress I ran into the same problem. Would be good if this change was documented in the Upgrade Guide since `interactiveLayerIds` wasn't required before for the `features` field to be present on the events.
